### PR TITLE
[WIP] Oleh rgb yuv

### PR DESCRIPTION
**Pull request by olegsemeniv**
Yuv to Rgb/ Rgb to Yuv both cuda and cpu implementation, and testing

### CODE REVIEW

File: libnd4j/include/ops/declarable/generic/images/yuvToRgb.cpp [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+#include <ops/declarable/headers/images.h>
+#include <ops/declarable/CustomOperations.h>  
+#include <helpers/ConstantTadHelper.h>
+#include <execution/Threads.h>
+
+namespace nd4j {
+namespace ops {
+
+CONFIGURABLE_OP_IMPL(yuv_to_rgb, 1, 1, false, 0, 0) {
```
Comment (raver119) [inline libnd4j/include/ops/declarable/generic/images/yuvToRgb.cpp]: This operation can be used inplace as far as i understand. So, why "false" here?

File: libnd4j/include/ops/declarable/headers/images.h [hunk 2]
```diff
@@ -65,6 +65,28 @@ namespace ops {
...
 
+    /**
+     * Rgb To Yuv
+     * Input arrays:
+     * 0 - input array with rank >= 1, must have at least one dimension equal 3, that is dimension containing channels.
+     * Int arguments:
+     * 0 - optional argument, corresponds to dimension with 3 channels
+     */
+#if NOT_EXCLUDED(OP_rgb_to_yuv)
+    DECLARE_CONFIGURABLE_OP(rgb_to_yuv, 1, 1, false, 0, 0);
```
Comment (raver119) [inline libnd4j/include/ops/declarable/headers/images.h]: same as above

File: libnd4j/include/ops/declarable/headers/images.h [hunk 3]
```diff
@@ -65,6 +65,28 @@ namespace ops {
...
+
+    /**
+     * Yuv To Rgb
+     * Input arrays:
+     * 0 - input array with rank >= 1, must have at least one dimension equal 3, that is dimension containing channels.
+     * Int arguments:
+     * 0 - optional argument, corresponds to dimension with 3 channels
+     */
+#if NOT_EXCLUDED(OP_rgb_to_yuv)
+    DECLARE_CONFIGURABLE_OP(yuv_to_rgb, 1, 1, false, 0, 0);
```
Comment (raver119) [inline libnd4j/include/ops/declarable/headers/images.h]: same as above

**Review by raver119 — CHANGES_REQUESTED** [review]
LGTM, minor tweak required

**Review by raver119 — DISMISSED** [review]

**Review by shyrma — DISMISSED** [review]

**Review by raver119 — APPROVED** [review]

**Pull request merged**